webhooks-extension/docs/InstallNightlyBuild.md

```diff
@@ -0,0 +1,58 @@
+# Nightly builds
+<br/>
+<br/>
+
+The Tekton Webhooks Extension has a hosted image of the latest builds located at gcr.io/tekton-nightly/extension
+<br/>
+<br/>
+
+* **To install the latest nightly driver:**
```

Comment:
I'd say image here and not driver, don't think anybody else uses the term `driver` in the knative/istio/tekton sense so it's a bit weird to start now

Reply:
ok